Automattic/woocommerce-payments
# fix: reset modal state on close

Fixes #

#### Changes proposed in this Pull Request









When the "Add payment methods" modal is closed, its state should be reset.

Before:
![2021-07-16 10 09 33](https://user-images.githubusercontent.com/273592/125969596-02658f4b-1fee-4bf1-9688-28d39ecbef73.gif)

After:
![2021-07-16 10 10 09](https://user-images.githubusercontent.com/273592/125969610-6b9c3def-f916-465d-b8d0-6249b5c1e686.gif)


#### Testing instructions





- Ensure you have "Enable UPE settings changes" enabled
- Ensure you have "Enable UPE checkout" enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Click the "Add payment method" button
- Select one of the available payment methods
- Close the modal (without adding)
- Re-open the modal
- The state should be reset

-------------------
